src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import { Button } from "./Button";
+import { FiArrowUpRight } from "react-icons/fi";
+export default { title: "Components/Button", component: Button };
+
+const args = {
+  onClick: () => {
+    console.log("clicked");
+  },
+  iconStart: <FiArrowUpRight />,
+  iconEnd: <FiArrowUpRight />,
+};
+
+export const _PrimaryDefault = {
+  render: () => <Button {...args}>Button 1</Button>,
```

Comment:
Use the name of the button instead here: Primary, Secondary, etc...


```suggestion
  render: () => <Button {...args}>Primary</Button>,
```

Reply:
Indeed! It is more logical.